plugins/action_trim_responses/src/trimResponsesWithoutDatabase.test.ts

```diff
@@ -1,7 +1,7 @@
 // Test suite for the testResponses Action.
 
 import DBConnect from '../../../src/components/databaseConnect'
-import { ActionQueueStatus, IsReviewableStatus } from '../../../src/generated/graphql'
+import { ActionQueueStatus, IsReviewable } from '../../../src/generated/graphql'
```

Comment:
I think this is still wrong - same for `graphql` generated file... Seems to be getting things from not the current changes in latest build (I'm comparing with the one in tag `v0.4.6-7`

Reply:
Oh yeah, should be "Reviewability" -- I did have this changed, but think I reverted accidentally. 🙄

Thanks 68677bf4